07LessonSeven/homework/index.html

```diff
@@ -1,10 +1,43 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Profession Portal</title>
   <link rel="stylesheet" href="css/style.css">
+  <link rel="stylesheet" href="CSS/style.css">
 </head>
 <body>
+  <header>
+    <h1>Profession Portal</h1>
+    <div>
+    <div class="main">
+
+      <div class="a">
+        <ul>
+          <li>Home</li>
+          <li>Companies</li>
+          <li>Positions</li>
+        </ul>
+      </div>
+    </header>
+</section>
+</div>
+  <div class="parent-container">
+    <section id="boxleft">
+      <h2>Hire an employee</h2>
+      <p>Vivamus congue rhoncus sem et placerat. Fusce nec nunc lobortis lorem ultrices facilisis. Ut dapibus blandit nunc, et consectetur dui.</p>
+        <span class="p2">Sign up as company</span>
+    </section>
+
+    <section id="boxright">
+      <div id="center">
+      <h2>Looking for a job</h2>
+      <p>Vivamus congue rhoncus sem et placerat. Fusce nec nunc lobortis lorem ultrices facilisis. Ut dapibus blandit nunc, et consectetur dui.</p>
+        <span class="p2">Sign up as employee</span>
+      </section>
+  </<div>
 
 </body>
 </html>
```

Comment:
Overall, great job on the assignment! I would say you still need to focus on HTML syntax and code readability but that will come with time and practice. Keep up the good work Khadijah.